src/pytorch_lightning/loggers/neptune.py

```diff
@@ -334,7 +335,7 @@ def _neptune_init_args(self):
 
         return args
 
-    def _construct_path_with_prefix(self, *keys) -> str:
+    def _construct_path_with_prefix(self, *keys: str) -> str:
```

Comment:
each call to this in the call hierarchy used a string. sanity checked the unpacking works with a str.

```python
LOGGER_JOIN_CHAR = "/"

def _construct_path_with_prefix(prefix, *keys: str) -> str:
    if prefix:
        return LOGGER_JOIN_CHAR.join([prefix, *keys])
    return LOGGER_JOIN_CHAR.join(keys)

_construct_path_with_prefix(None, "zed/quu")
Out: 'zed/quu'

_construct_path_with_prefix("root", "zed/quu")
Out: 'root/zed/quu'
```